### DISCUSSION TITLE
`Alt + Click` to open file

### DISCUSSION THREAD

**Discussion by brettcannon** [microsoft/pylance-release#2670] [Enhancement]
### Discussed in https://github.com/microsoft/vscode-python/discussions/16666

<div type='discussions-op-text'>

<sup>Originally posted by **beasthunter97** July 13, 2021</sup>
Currently, I can only open file if the importing file is available or open a file with prefix `file:///` using `Alt + Click`.
It would be very nice if I'm able to open and create file/folder using when importing (if the importing file doesn't exist, new file is created) and string editing (when open command for example) with `Alt + Click`.
Thank you very much!</div>

**Comment by judej** [comment]
@beasthunter97, could you please clarify the enhancement you are looking for? is the scenario:
- write the import statement in the file
- Alt-click the module name - which results in the file being created?
thanks!

**Comment by anhq-nguyen** [comment]
Yes, for example: 
`import test`
When I use `Alt + Click` on `test`, I expect it to:

- Open file `test.py` if it exist (which is already working)
- Create `test.py` file if it doesn't exist

and the same with string `"./abc.py"`

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
